js/libs/form.js

```
@@ -2278,16 +2268,16 @@ Fliplet().then(async function() {
                       return { id: option };
                     });
 
-                    if (!_.isEmpty(field.value)) {
+                    if (!Fliplet.FormBuilderUtils.isEmpty(field.value)) {
                       switch (field._type) {
                         case 'flCheckbox':
-                          var selectedValues = _.difference(field.value, values);
+                          var selectedValues = Fliplet.FormBuilderUtils.difference(field.value, values);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)